### PR TITLE
Added the function that uptades the clock

### PR DESCRIPTION
**Pull request by LuiAlv2502**
In this commit I added a function that makes a clock work

### CODE REVIEW

File: tix_clock/tix_clock.ino [hunk 1]
```diff
@@ -75,3 +76,23 @@ int GiveMeRandomNumber(int lowerLimit, int upperLimit)
...
+    if(minute == 60)
+    {
+        minute = 0;
+        hour = hour + 1;  
+    }  
+    if (hour == 25)
+    {
+        hour = 0;  
+    }
+
```
Comment (caralvar) [inline tix_clock/tix_clock.ino]: please remove this empty line

**Review by caralvar — CHANGES_REQUESTED** [review]

**Comment by LuiAlv2502** [comment]
eliminated the blank space in the line 97

File: tix_clock/tix_clock.ino (hunk 1 above)
Comment (LuiAlv2502) [reply tix_clock/tix_clock.ino]: eliminated the blank space in the line 97

File: tix_clock/tix_clock.ino [hunk 2]
```diff
@@ -75,3 +76,22 @@ int GiveMeRandomNumber(int lowerLimit, int upperLimit)
...
+        second = 0;
+        minute = minute + 1;
+        
+    }  
+    if(minute == 60)
+    {
+        minute = 0;
+        hour = hour + 1;  
+    }  
+    if (hour == 25)
```
Comment (caralvar) [inline tix_clock/tix_clock.ino]: Please change this to 24, because there are only 24 hours in the day :)

**Review by caralvar — CHANGES_REQUESTED** [review]

File: tix_clock/tix_clock.ino (hunk 2 above)
Comment (LuiAlv2502) [reply tix_clock/tix_clock.ino]: changed it to 24 in the new commit

**Pull request merged**